src/nodes/accessors/MaterialNode.js

```diff
@@ -235,6 +235,12 @@ class MaterialNode extends Node {
 				node = normalMap( this.getTexture( 'normal' ), this.getCache( 'normalScale', 'vec2' ) );
 				node.normalMapType = material.normalMapType;
 
+				if ( material.normalMap.userData && material.normalMap.userData.unpackNormalMode ) {
+
+					node.unpackNormalMode = material.normalMap.userData.unpackNormalMode;
```

Comment:
I have reconsidered this bit. Do you mind changing this check by explicitly testing for compatible texture formats? In this way, we don't have to store data in `Texture.userData` which means if loaders or apps properly configure compressed normals maps they will automatically work with `WebGPURenderer`.

Reply:
OK. Done!